.github/workflows/maven.yml

```diff
@@ -11,9 +11,18 @@ jobs:
   build:
     strategy:
       matrix:
-        java: [1.8, 11]
+        java: [1.8, 11, 17]
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.6', '3.7', '3.8', '3.9']
+        python-version: ['3.7', '3.8', '3.9']
```

Comment:
I assume it's not overly relevant for this repo, but soon I could imagine adding 3.10 as well.

Reply:
All fine with me,  shifting the supported Python version range would go in pair with the dropping of EOL Python 3.6. Feel free to open a PR to that effect